docs/3.x/tech/installation-guide/database.adoc

```diff
@@ -95,6 +95,6 @@ $ sudo apt-get install postgresql-server
 $ sudo apt-get install postgresql-contrib
```

Comment:
1. Missed `Mac OSX` on line 41. 
2. Is the download link still good? to the MySQL downloads?